package.json

```diff
@@ -22,7 +22,7 @@
         "express": "~4.16.0",
         "express-jwt": "^5.3.1",
         "express-jwt-authz": "^2.3.1",
-        "express-openapi-validator": "^0.53.3",
+        "express-openapi-validator": "git+https://github.com/webstacker/express-openapi-validator.git",
```

Comment:
Should this be forked from CICA?

Reply:
I've got an open issue with the authors at the moment, hence the fork is on my account. Don't think it matters for the moment. Depending on their response we might need a permanent from here, otherwise we can ditch this and use their updated version from npm.

Reply:
My thinking was that if the fix needed to be a permanent fork, then it would be be most fitting to have it in the CICA GitHub account. Use this at the moment, and we can change it if needed at a later date!

Reply:
👍